relay-server/src/services/buffer/mod.rs

```diff
@@ -220,11 +220,8 @@ impl EnvelopeBufferService {
         services: &Services,
         envelopes_tx_permit: Permit<'a, DequeuedEnvelope>,
     ) -> Result<Duration, EnvelopeBufferError> {
-        relay_log::trace!("EnvelopeBufferService: peeking the buffer");
```

Comment:
Changes in the file are unrelated, I got irritated by the amount of log lines I saw during integration testing.